sql/common/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/raster/RasterConstructors.scala

```diff
@@ -82,7 +82,40 @@ case class RS_FromGeoTiff(inputExpressions: Seq[Expression]) extends Expression
     if (bytes == null) {
       null
     } else {
-      Constructors.fromGeoTiff(bytes)
+      RasterConstructors.fromGeoTiff(bytes)
     }
   }
 }
+
+case class RS_MakeEmptyRaster(inputExpressions: Seq[Expression]) extends Expression with CodegenFallback
+  with ExpectsInputTypes with SerdeAware {
```

Comment:
Mixin the trait `ImplicitCastInputTypes` instead of `ExpectsInputTypes` makes it more friendly for users, otherwise user must write `RS_MakeEmptyRaster(100, 80, 5.0, 4.0, 0.4)` instead of `RS_MakeEmptyRaster(100, 80, 5, 4, 0.4)`.